events/button.js

```diff
@@ -10,19 +10,15 @@ module.exports = (client) => {
         if (!interaction.isMessageComponent() && interaction.componentType !== "BUTTON") return
         console.log(interaction.customId)
         if (interaction.customId == "igjoin") {
-            //if (db.get("started") == "yes") return interaction.reply(`The game has already started!`, { ephemeral: true })
+            interaction.deferUpdate()
             let guy = interaction.member
             if (guy.roles.cache.has(ids.spectator)) guy.roles.remove(ids.spectator) //spec
             if (guy.roles.cache.has(ids.narrator)) guy.roles.remove(ids.narrator) //narr
             if (guy.roles.cache.has(ids.mini)) guy.roles.remove(ids.mini) //mininarr
             if (guy.roles.cache.has(ids.jowov)) guy.roles.remove(ids.jowov) //jowov
             let role = interaction.guild.roles.cache.get(ids.alive)
-            await guy.roles
-                .add(ids.alive)
-                .then((g) => g.setNickname(role.members.size.toString()).catch((e) => interaction.reply(`Error: ${e.message}`)))
-                .catch((e) => interaction.reply(`Error: ${e.message}`))
+            await guy.roles.add(ids.alive).catch((e) => interaction.reply(`Error: ${e.message}`))
```

Comment:
```suggestion
            await guy.roles.add(ids.alive).catch((e) => interaction.editReply(`Error: ${e.message}`))
```
deferred replies can only be edited